legacy/build-deploy-docker-compose.sh

```diff
@@ -1303,8 +1331,6 @@ do
 
   SERVICE_NAME=${DBAAS_ENTRY_SPLIT[0]}
   SERVICE_TYPE=${DBAAS_ENTRY_SPLIT[1]}
-  # remove the image from images to pull
-  unset IMAGES_PULL[$SERVICE_NAME]
```

Comment:
removed from here, because it should be resolved by handling the dbaas checks in the additional code

Reply:
correct!